src/test/java/com/uid2/admin/vertx/SiteServiceTest.java

```diff
@@ -26,7 +26,7 @@
 public class SiteServiceTest extends ServiceTestBase {
     @Override
     protected IService createService() {
-        return new SiteService(auth, writeLock, storeWriter, siteProvider, clientKeyProvider);
+        return new SiteService(auth, writeLock, storeWriter, siteProvider, legacyClientKeyProvider);
```

Comment:
I think you can update `ServiceTestBase.setClientKeys()` to remove references to `clientKeyProvider` and make it unambigous that only LegacyClientKeyProvider is used

Reply:
Let me do this in my own PR